vms/platformvm/vm_test.go

```diff
@@ -277,6 +277,9 @@ func defaultVM(t *testing.T, f upgradetest.Fork) (*VM, *txstest.WalletFactory, d
 
 	// align chain time and local clock
 	vm.state.SetTimestamp(vm.clock.Time())
+	vm.state.SetFeeState(feecomponent.State{
+		Capacity: defaultDynamicFeeConfig.MaxGasCapacity,
+	})
```

Comment:
Because we manually set the timestamp here - the initial capacity would be `0`. So this enables transactions to be issued.